### PR TITLE
Do not merge: add new terms/synonyms and xrefs related to COVID-19

### LINKED ISSUE #1216: add synonym to '2019 novel coronavirus infection'
**Mondo term (ID and Label)**
'2019 novel coronavirus infection'

**Synonym to be added**
SARS-CoV-2
dbxref:doi:10.1038/s41564-020-0695-z

**Synonym subtype: ie, broad/exact/narrow/related**
exact

- [x] fix typo in other doi

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1216

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -428453,12 +428453,20 @@ created_by: http://orcid.org/0000-0001-5208-3432
 id: MONDO:0100096
 name: 2019 novel coronavirus infection
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: the official WHO name is COVID-19

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -428453,12 +428453,20 @@ created_by: http://orcid.org/0000-0001-5208-3432
...
 def: "Coronavirus disease 2019 (COVID-19) is a respiratory illness that can spread from person to person. The virus that causes COVID-19 is a novel coronavirus that was first identified during an investigation into an outbreak in Wuhan, China." [https://www.cdc.gov/coronavirus/2019-ncov/downloads/2019-ncov-factsheet.pdf]
+synonym: "2019-nCoV (novel coronavirus) detected" RELATED [SCTID:1240581000000104]
 synonym: "2019-nCoV infection" EXACT [https://www.cdc.gov/coronavirus/2019-ncov/index.html]
 synonym: "COVID-19" EXACT [https://www.cdc.gov/coronavirus/2019-ncov/about/index.html]
+synonym: "disease caused by 2019-nCoV (novel coronavirus)" EXACT [] {comment="SCTID:1240751000000100"}
 synonym: "SARS-CoV-2" EXACT [doi:10.1038/s41564-020-0695-z]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: our DPs say "% infection"

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: I relabeled this in a separate PR #1340

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: We do have % infection as synonyms.

I'm unclear what the action item here is - do we not want this synonym?

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: From our conversation on Skype - my understanding is we do not want to add these synonyms, we just want the xrefs.

**Comment by nicolevasilevsky** [comment]
Closing this. I added the xrefs on a separate PR #1352

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -428453,12 +428453,20 @@ created_by: http://orcid.org/0000-0001-5208-3432
...
 synonym: "severe acute respiratory syndrome coronavirus 2" EXACT [doi:10.1038/s41564-020-0695-z]
 xref: DOID:0080600 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005719 ! coronavinae infectious disease
+xref: ICD10:U07.1 {source="MONDO:relatedTo"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:428463]: is this not exact?

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:428463]: no, Coronavirinae is a parent: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=2697049

**Pull request closed without merging**